src/reference/informative/informC04_component_reference.rst

```diff
@@ -8,4 +8,197 @@
 
   .. container:: infospec
 
-    **TODO**
+    .. container:: heading3
+
+      Understanding component references
+
+    As with references to units and variables, the term "component reference" refers to places where you need to specify or reference a component using its name (but not assign it its name).
```

Comment:
I'd drop "specify or" and "(but not assign its name)"

Maybe "reference an existing component by its name" ?

Reply:
Agree with @MichaelClerx's remark and suggestion.

Reply:
I would also remove "(but not assign it its name)" as this wouldn't be needed anymore if we were to go with @MichaelClerx's suggestion.

Reply:
Fixed in ddfc79d